v-next/hardhat/test/internal/builtin-plugins/network-manager/network-manager.ts

```diff
@@ -302,23 +302,6 @@ describe("NetworkManagerImplementation", () => {
       );
     });
 
-    it("should throw an error if the specified network config override has mixed properties from http and edr networks", async () => {
```

Comment:
Because we now allow arbitrary properties, we can no longer provide this level of helpful error message.